pkg/api/api.go

```diff
@@ -89,6 +90,11 @@ type ApiError struct {
 	Err error
 }
 
+const (
+	// tenantIDKey is the key that holds the tenant ID in a request context.
+	tenantIDKey string = "tenantID"
```

Comment:
If we want only this package to be able to edit this, let's use `tenantIDKey = struct{}` and use only this variable